### PR TITLE
Introduce docker file and pipeline for creating image with bosh cli.

### PR DESCRIPTION
**Pull request by karthijrk**
@d Please take a look when you have a change.

### CODE REVIEW

File: images/gpdb_perf/test.bash [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+it_has_ruby23() {
+  it_has_executable_version "ruby" "ruby 2.3"
+}
+
+it_has_executable_version() {
+  local executable=$1
+  local grep_string=$2
+  if [ "${grep_string}" == "$(${executable} --version | grep -o "${grep_string}")" ]; then
+    echo "${grep_string} found"
```
Comment (d) [inline images/gpdb_perf/test.bash]: This is unnecessarily complex. The following would do the same:
```bash
"${executable}" --version | grep "${grep_string}"
```

File: pipelines/qp_images.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+resources:
+  - name: qp-images
+    type: git
+    source:
+      branch: bosh
```
Comment (d) [inline pipelines/qp_images.yml]: branch should be master

**Review by d — CHANGES_REQUESTED** [review]

File: images/gpdb_perf/test.bash (hunk 1 above)
Comment (karthijrk) [reply images/gpdb_perf/test.bash]: cool. Let me update that.

File: pipelines/qp_images.yml (hunk 2 above)
Comment (karthijrk) [reply pipelines/qp_images.yml]: Yes, I will change this.

**Comment by karthijrk** [comment]
@d, I addressed your comments and here is ci link -  https://ci.orca.pivotalci.info/teams/main/pipelines/qp-images

File: pipelines/qp_images.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+resources:
+  - name: qp-images
+    type: git
+    source:
+      branch: master
+      uri: git@github.com:Pivotal-DataFabric/qp-images.git
+      private_key: {{qp_images_deploy_key}}
```
Comment (d) [inline pipelines/qp_images.yml:7]: I don't think we need a deploy key. This is a public repo. But we can ship it before changing this bit.

**Comment by d** [comment]
LGTM. Ship it!

File: pipelines/qp_images.yml (hunk 3 above)
Comment (karthijrk) [reply pipelines/qp_images.yml:7]: oh, I thought this is private. :)

**Pull request merged**